src/datatable/datatable.jl

```diff
@@ -602,7 +594,7 @@ Base.setindex!(dt::DataTable, v, ::Colon, col_inds) =
     (dt[col_inds] = v; dt)
 
 # Special deletion assignment
-Base.setindex!(dt::DataTable, x::Void, col_ind::Int) = delete!(dt, col_ind)
+Base.setindex!(dt::DataTable, x::Null, col_ind::Int) = delete!(dt, col_ind)
```

Comment:
I didn't realize we had this at all. 💯 for removing it in favor of an explicit function.